webapp/packages/core-root/src/SessionEventSource.ts

```diff
@@ -135,6 +137,26 @@ export class SessionEventSource implements IServerEventEmitter<ISessionEvent, IS
 
   disconnect() {
```

Comment:
https://rxjs.dev/api/webSocket/webSocket - the rxjs websocket is tried to reconnect cause there was some subscriptions